webview-ui/src/locales/ja/translation.json

```
@@ -0,0 +1,32 @@
+{
+	"announcement": {
+		"newInVersion": "バージョン{{version}}の新機能",
+		"checkpointsTitle": "チェックポイント",
+		"checkpointsDescription": "進捗を保存するためのチェックポイントを作成します。",
+		"compareTitle": "比較",
+		"compareDescription": "チェックポイント間の変更を比較します。",
+		"restoreTitle": "復元",
+		"restoreDescription": "以前のバージョンを復元します。",
+		"seeNewChangesTitle": "新しい変更を見る",
+		"seeNewChangesDescription": "最新の変更を表示します。",
+		"seeDemo": "デモを見る",
+		"joinOurCommunities": "最新情報を入手するには、 私たちの <DiscordLink>Discord</DiscordLink> または <RedditLink>Reddit</RedditLink> に参加してください ！"
```

I made some corrections to a part of the translation.

```suggestion
		"joinOurCommunities": "最新情報を入手するには、私たちの <DiscordLink>Discord</DiscordLink> または <RedditLink>Reddit</RedditLink> に参加してください！"
```